eng/Build.props

```diff
@@ -64,16 +64,20 @@
         <!-- Build the ANCM custom action -->
```

Comment:
Why restrict the `$(TargetArchitecture)` on the line above to `x86` or `x64`❔ Can these projects build on `arm64` now❔

Reply:
So add a 3rd OR for arm64?